src/GraphQL/Utility/FileUpload.php

```diff
@@ -259,6 +269,9 @@ public function saveFileUpload(UploadedFile $uploaded_file, array $settings): Fi
 
       // Validate against file_validate() first with the temporary path.
       $errors = file_validate($file, $validators);
+      $maxResolution = $settings['max_resolution'] ?? 0;
+      $minResolution = $settings['min_resolution'] ?? 0;
+      $errors += $this->validateFileImageResolution($file, $maxResolution, $minResolution);
```

Comment:
we should only invoke this method if we have the min/max resolution settings? That avoids calling the image factory with arbitrary files where we know those are not images.